src/redux/store.test.ts

```diff
@@ -25,6 +27,16 @@ const resetStateOnInvalidStoredAccount = jest.spyOn(
 
 const loggerErrorSpy = jest.spyOn(Logger, 'error')
 
+const getNonApiReducers = <R = Omit<RootState, ApiReducersKeys>>(state: RootState): R => {
```

Comment:
nit: prefer `function` instead of const arrow functions, as it's easier to spot it's a function when parsing the file visually. Plus it's a few less chars :D
```suggestion
function getNonApiReducers<R = Omit<RootState, ApiReducersKeys>>(state: RootState): R {
```

I know we haven't been super consistent with this in this code base though.